[{
    "cve": {
        "data_type": "CVE",
        "references": {
            "reference_data": [{
                "url": "http://docs.python-requests.org/en/master/community/updates/#release-and-version-history",
                "name": "http://docs.python-requests.org/en/master/community/updates/#release-and-version-history",
                "tags": ["Release Notes", "Third Party Advisory"],
                "refsource": "CONFIRM"
            }, {
                "url": "http://lists.opensuse.org/opensuse-security-announce/2019-07/msg00024.html",
                "name": "openSUSE-SU-2019:1754",
                "tags": [],
                "refsource": "SUSE"
            }, {
                "url": "https://access.redhat.com/errata/RHSA-2019:2035",
                "name": "RHSA-2019:2035",
                "tags": [],
                "refsource": "REDHAT"
            }, {
                "url": "https://bugs.debian.org/910766",
                "name": "https://bugs.debian.org/910766",
                "tags": ["Exploit", "Issue Tracking", "Patch", "Third Party Advisory"],
                "refsource": "MISC"
            }, {
                "url": "https://github.com/requests/requests/commit/c45d7c49ea75133e52ab22a8e9e13173938e36ff",
                "name": "https://github.com/requests/requests/commit/c45d7c49ea75133e52ab22a8e9e13173938e36ff",
                "tags": ["Patch", "Third Party Advisory"],
                "refsource": "MISC"
            }, {
                "url": "https://github.com/requests/requests/issues/4716",
                "name": "https://github.com/requests/requests/issues/4716",
                "tags": ["Exploit", "Patch", "Third Party Advisory"],
                "refsource": "MISC"
            }, {
                "url": "https://github.com/requests/requests/pull/4718",
                "name": "https://github.com/requests/requests/pull/4718",
                "tags": ["Patch", "Third Party Advisory"],
                "refsource": "MISC"
            }, {
                "url": "https://usn.ubuntu.com/3790-1/",
                "name": "USN-3790-1",
                "tags": ["Third Party Advisory"],
                "refsource": "UBUNTU"
            }, {
                "url": "https://usn.ubuntu.com/3790-2/",
                "name": "USN-3790-2",
                "tags": ["Third Party Advisory"],
                "refsource": "UBUNTU"
            }]
        },
        "data_format": "MITRE",
        "description": {
            "description_data": [{
                "lang": "en",
                "value": "The Requests package before 2.20.0 for Python sends an HTTP Authorization header to an http URI upon receiving a same-hostname https-to-http redirect, which makes it easier for remote attackers to discover credentials by sniffing the network."
            }]
        },
        "problemtype": {
            "problemtype_data": [{
                "description": [{
                    "lang": "en",
                    "value": "CWE-522"
                }]
            }]
        },
        "data_version": "4.0",
        "CVE_data_meta": {
            "ID": "CVE-2018-18074",
            "ASSIGNER": "cve@mitre.org"
        }
    },
    "impact": {
        "baseMetricV2": {
            "cvssV2": {
                "version": "2.0",
                "baseScore": 5.0,
                "accessVector": "NETWORK",
                "vectorString": "AV:N/AC:L/Au:N/C:P/I:N/A:N",
                "authentication": "NONE",
                "integrityImpact": "NONE",
                "accessComplexity": "LOW",
                "availabilityImpact": "NONE",
                "confidentialityImpact": "PARTIAL"
            },
            "severity": "MEDIUM",
            "impactScore": 2.9,
            "obtainAllPrivilege": false,
            "exploitabilityScore": 10.0,
            "obtainUserPrivilege": false,
            "obtainOtherPrivilege": false,
            "userInteractionRequired": false
        },
        "baseMetricV3": {
            "cvssV3": {
                "scope": "UNCHANGED",
                "version": "3.0",
                "baseScore": 9.8,
                "attackVector": "NETWORK",
                "baseSeverity": "CRITICAL",
                "vectorString": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
                "integrityImpact": "HIGH",
                "userInteraction": "NONE",
                "attackComplexity": "LOW",
                "availabilityImpact": "HIGH",
                "privilegesRequired": "NONE",
                "confidentialityImpact": "HIGH"
            },
            "impactScore": 5.9,
            "exploitabilityScore": 3.9
        }
    },
    "publishedDate": "2018-10-09T17:29Z",
    "configurations": {
        "nodes": [{
            "operator": "OR",
            "cpe_match": [{
                "cpe23Uri": "cpe:2.3:a:opencveio:opencve:1:*:*:*:*:*:*:*",
                "vulnerable": true
            }, {
                "cpe23Uri": "cpe:2.3:a:opencveio:opencve:2:*:*:*:*:*:*:*",
                "vulnerable": true
            }]
        }, {
            "operator": "OR",
            "cpe_match": [{
                "cpe23Uri": "cpe:2.3:o:canonical:ubuntu_linux:14.04:*:*:*:lts:*:*:*",
                "vulnerable": true
            }, {
                "cpe23Uri": "cpe:2.3:o:canonical:ubuntu_linux:16.04:*:*:*:lts:*:*:*",
                "vulnerable": true
            }, {
                "cpe23Uri": "cpe:2.3:o:canonical:ubuntu_linux:18.04:*:*:*:lts:*:*:*",
                "vulnerable": true
            }, {
                "cpe23Uri": "cpe:2.3:o:canonical:ubuntu_linux:18.10:*:*:*:*:*:*:*",
                "vulnerable": true
            }]
        }],
        "CVE_data_version": "4.0"
    },
    "lastModifiedDate": "2021-12-21T13:37Z"
}]